Source/Core/VideoCommon/DriverDetails.h

```diff
@@ -195,6 +195,18 @@ namespace DriverDetails
 		// Mesa introduced geometry shader support for radeon and sandy bridge devices and failed to test it with us.
 		// Causes misrenderings on a large amount of things that draw lines.
 		BUG_BROKENGEOMETRYSHADERS,
+
+		// Bug: glMapBuffer on the SSBO used for bounding box causes significant slowdown
```

Comment:
Please summarize what this flag is used for on AMD, not why the workaround is slow on Nvidia.